mobile/ios/scripts/build_arch.sh

```diff
@@ -27,7 +35,7 @@ export CFLAGS="-isysroot ${DEVELOPER}/Platforms/${PLATFORM}.platform/Developer/S
 export CXXFLAGS="-isysroot ${DEVELOPER}/Platforms/${PLATFORM}.platform/Developer/SDKs/${PLATFORM}${SDKVERSION}.sdk"
 
 (
-    cd ../../
+    cd $SOURCEDIR
     test -x ./configure || autoreconf -i
     ./configure --disable-shared \
```

Comment:
I'd add `-q` to `./configure` such that less output is produced. This would allow us to focus on errors only when something goes wrong and we avoid bloating the developer with low-level details.